.gitignore

```diff
@@ -62,6 +62,9 @@ target/
 *~
 \#*\#
 
+#vscode
+.vscode/
```

Comment:
I believe `.vscode` is intended to be committed to source control.  I currently don't use it as I'm waiting for serpent support (someone is working on it), but once we do we will want this directory committed I think.